identity_enforcer.py

```diff
@@ -0,0 +1,38 @@
+import discord
+import re
+from channels import DRONE_HIVE_CHANNELS
+from roles import DRONE, ENFORCER_DRONE, has_role
+from bot_utils import get_id
+
+class Identity_Enforcer():
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.on_message = [self.enforce_identity]
+        self.on_ready = [self.report_online]
+        self.channels = [] + DRONE_HIVE_CHANNELS
```

Comment:
```suggestion
        self.channels = [DRONE_HIVE_CHANNELS]
```

Reply:
DRONE_HIVE_CHANNELS is a list, and python gets fucky about traversing a list in a list, `[] + LIST` is similar to .extend() in functionality

Reply:
Oh, now I get it. Had not read it properly. But then simply `self.channels = DRONE_HIVE_CHANNELS` might be a good idea.

Reply:
Good point, will update <3